match/lib/match/storage/google_cloud_storage.rb

```diff
@@ -349,7 +349,7 @@ def check_bucket_permissions
           UI.message("\t\t 1. Click on your bucket to open it".cyan)
           UI.message("\t\t 2. Click 'Permissions'".cyan)
           UI.message("\t\t 3. Click 'Add members'".cyan)
-          UI.message("\t\t 4. Enter the email of your service account".cyan)
+          UI.message("\t\t 4. Enter the *email of your service account* (not your own email)".cyan)
```

Comment:
I think we need to remove the `*` here as they don't bold anything and nothing like this is used anywhere else in fastlane. If anything would suggest we make this change 👇 
```suggestion
          UI.message("\t\t 4. Enter the service account email address".cyan)
```